### DISCUSSION TITLE
【Demo及附件】如何控制文本的宽度？用户名字过长会无限延长

### DISCUSSION THREAD

**Discussion by mianma12138** [Tencent/libpag#2358] [Q&A] [answered]
如何控制文本的宽度？用户名字过长会无限延长，设计控制还是开发可以控制？
![微信截图_20240702185904](https://github.com/Tencent/libpag/assets/92774467/6f28a728-8b25-4813-9fe9-caa0cb6ce50a)

**Comment by kevingpqi123** [comment, marked as answer]
让你们的设计师设计的时候控制下文本图层的长度，和移动端开发的文本控件一样，文本显示的区域不会超过文本控件的范围

**Reply by mianma12138** [reply]
看了下官方提供的文档，设计师控制文本框宽度，但是名字过长，文字在框里就会自适应缩小。。。。

**Reply by mianma12138** [reply]
采用文本框会出现这么几个问题。pag该如何解决，只在第一行正确显示
![微信截图_20240703141025](https://github.com/Tencent/libpag/assets/92774467/9fd94046-e6fa-4ce1-9b61-da84b3819a51)
![Uploading Snipaste_2024-07-03_14-08-01.png…]()

**Reply by kevingpqi123** [reply]
AE 中文本分为两类：框文本和点文本，点文本不会换行和缩小字体
